ThePensionsRegulator.Frontend/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using GovUk.Frontend.AspNetCore;
+using GovUk.Frontend.AspNetCore.Extensions;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace ThePensionsRegulator.Frontend
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddTprFrontend(this IServiceCollection services)
+        {
+            return services.AddTprFrontendExtensions(options => { options.AddImportsToHtml = false; });
+        }
+
+        public static IServiceCollection AddTprFrontendExtensions(
+            this IServiceCollection services,
+            Action<GovUkFrontendAspNetCoreOptions> options)
+        {
+            if (services == null)
+            {
+                throw new ArgumentNullException(nameof(services));
+            }
+
+            services.AddGovUkFrontendExtensions(options);
```

Comment:
```suggestion
            services.AddTprFrontendExtensions(options);
```

Should this not be `AddTprFrontend`?

Reply:
No it should be how it is, misunderstood so marking as resolved